apps/test/netsim/NetSimRouterNode.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 /* global describe, beforeEach, it */
 
-var _ = require('lodash');
 var testUtils = require('../util/testUtils');
 var NetSimTestUtils = require('../util/netsimTestUtils');
-var utils = require('@cdo/apps/utils');
+var _ = require('@cdo/apps/lodash');
+var utils = require('@cdo/apps/utils'); // Provides String.prototype.repeat
```

Comment:
Though not strictly necessary, it seemed like a good idea to be consistent and use the copy of lodash from apps instead of also including the full module in tests.